src/main/java/com/bandwidth/iris/sdk/model/AvailableNpaNxx.java

```diff
@@ -28,7 +28,7 @@
     @XmlElement(name = "Quantity")
     private int quantity;
     @XmlElement(name = "State")
-    private int state;
+    private string state;
```

Comment:
It is incorrect type.

Reply:
@v-ladynev it should be a string - the API accepts a 2 digit state code

for ex = "NC" or "NY" or "CA" would be acceptable values

The [C# SDK](https://github.com/Bandwidth/csharp-bandwidth-iris/blob/d482ee7f65fdd1d517a3f2e83c37f432cefbcfb9/Bandwidth.Iris/Model/AvailableNpaNxx.cs#L21) also declares this value as a string

We are checking the release pipeline and should have this published as v1.9 shortly